tests/unittests/core/test_trial.py

```diff
@@ -120,12 +120,12 @@ def test_value_equal(self, exp_config):
         assert trials[0].params[0] == Trial.Param(**exp_config[1][0]['params'][0])
         assert trials[0].params[1] != Trial.Param(**exp_config[1][0]['params'][0])
 
-    def test_str_trial(self, exp_config):
+    def test_pfmt_trial(self, exp_config):
         """Test representation of `Trial`."""
         t = Trial(**exp_config[1][1])
-        assert str(t) == "Trial(experiment='supernaedo2', status='completed',\n"\
-                         "      params=/decoding_layer:lstm_with_attention\n"\
-                         "             /encoding_layer:gru)"
+        assert t.pfmt() == "Trial(experiment='supernaedo2', status='completed',\n"\
```

Comment:
Could we adapt the test instead and drop pfmt?

Reply:
is there really a need for testing printing ?
How hard of a spec is this ?
Like do we expect people relying on it to parse Trials ?

Reply:
I agree that isn't critical. Feel free to drop this one as well as pfmt.